py3status/modules/selinux.py

```diff
@@ -36,26 +41,31 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     format = 'selinux: {state}'
+    string_disabled = 'disabled'
+    string_enforcing = 'enforcing'
+    string_permissive = 'permissive'
 
-    def selinux_status(self):
+    def post_config_hook(self):
+        if not self.py3.check_commands('getenforce'):
+            return {'cache_until': self.py3.CACHE_FOREVER,
+                    'color': self.py3.COLOR_BAD,
+                    'full_text': string_unavailable}
```

Comment:
As elsewhere this is not how `post_config_hook()` works.  Sometimes it is good to just do one PR first so you get to know how it all works and we can fix things up as needed.  Currently you are creating work for both of us ;)